services/src/main/java/io/druid/cli/CoordinatorJettyServerInitializer.java

```diff
@@ -62,7 +62,9 @@
       "/fonts/*",
       "/old-console/*",
       "/coordinator/false",
-      "/overlord/false"
+      "/overlord/false",
+      "/health",
+      "/druid/coordinator/v1/isLeader"
```

Comment:
I think you need to include the overlord isLeader URL too.

Reply:
Instead of putting `/health` and `/druid/coordinator/v1/isLeader` here (& similar in other CLI files) I suggest adding something in the resource itself that just sets the "yeah, I checked myself" flag. Probably using a new helper in AuthorizationUtils. It's better because it's more local -- it's really easy for someone reviewing the http code to miss that the security check is skipped here.

Reply:
I could move the authorization check into the resource, but to skip authentication I'd still need to setup the UnsecuredResourceFilter with the unsecured paths (since the authentication checks happen before the request hits the resource handler methods).

Maybe another approach for authentication could be to have the resources with unsecured endpoints register with some singleton injected registry object that UnsecuredResourceFilter checks (and have UnsecuredResourceFilter apply to all paths), but I'd rather handle that in a separate approach.

Reply:
Ok, that makes sense. In that case I'd suggest leaving things as you have them here, but adding a comment in the resource file saying that _this_ file has it listed as unsecured.

Really, I'm just concerned that a reader of the http code is going to be confused about where the security check is. I was suggesting a programmatic approach but a comment works too.

Reply:
Added comments on the resources with these unsecured paths, pointing to the service initialization code